widgets/labels/src/LabelsTable.jsx

```diff
@@ -51,6 +47,12 @@ export default function LabelsTable({ data, toolbox }) {
 
     const tdStyle = { textOverflow: 'ellipsis', overflow: 'hidden' };
 
+    const currentLabelAlreadyUsed = !!_.find(
```

Comment:
A bit shorter :slightly_smiling_face: 

```suggestion
    const currentLabelAlreadyUsed = _.includes(
```

Reply:
See my previous comment about `_.includes`.